liquid-rust-driver/src/lower/basic_block.rs

```diff
@@ -9,7 +8,7 @@ impl<'tcx> Lower<'tcx> for mir::BasicBlock {
     type Output = BasicBlock;
 
     fn lower(&self, _lcx: LowerCtx<'tcx>) -> LowerResult<Self::Output> {
-        Ok(BasicBlock::new(self.index()))
+        Ok(*self)
```

Comment:
I think we can just skip the `impl Lower for BasicBlock` altogether, no?

Reply:
yes totally.